contract/test/test-coveredCall.js

```diff
@@ -100,10 +104,11 @@ test('contract with valid offers', async t => {
   const bobInvitation = await E(aliceSeat).getOfferResult();
   t.truthy(await E(invitationIssuer).isLive(bobInvitation));
 
+  const recoveryPurse = E(invitationIssuer).makeEmptyPurse();
   // Bob receives the invitation from Alice
   // He doesn't trust alice, so he wants inspect the invitation
   // claimedInvitation === option
-  const claimedInvitation = await E(invitationIssuer).claim(bobInvitation);
+  const claimedInvitation = await claim(recoveryPurse, bobInvitation);
```

Comment:
Just do manually what the `claim` helper does automatically -- deposit into an invitation purse. This would also return the amount, which could be used instead of the following `getInvitationDetails` call.

Reply:
If this pattern becomes common, we could consider making the `claim` helper more official rather than itself deprecated. We'll see.